include/lsst/afw/geom/Transform.h

```diff
@@ -88,34 +86,29 @@ class Transform final :
     /**
      * Construct a Transform from a deep copy of an ast::Mapping
      *
-     * The internal FrameSet consists of a frame constructed by each endpoint
-     * connected by the mapping.
-     *
      * @param[in] mapping  ast::Mapping describing the desired transformation
      * @param[in] simplify  Simplify the mapping? This combines component mappings
-     *     and removes redundant components where possible.
+     *   where it possible to do so without affecting accuracy.
      */
     explicit Transform(ast::Mapping const &mapping, bool simplify = true);
 
     /**
-     * Construct a Transform from a deep copy of a FrameSet.
+     * Construct a Transform from a deep copy of a ast::FrameSet
      *
-     * The result transforms from the "base" frame to the "current" frame.
+     * The result transforms from the "base" frame to the "current" frame of the provided FrameSet.
      * The "from" endpoint is used to normalize the "base" frame
      * and the "to" endpoint is used to normalize the "current" frame.
      *
      * This is pickier than the constructor that takes an ast::Mapping in that:
      * - SpherePointEndpoint must be associated with an ast::SkyFrame and the SkyFrame axes
-     *   are swapped if necessary to the standard order: longitude, latitude.
+     *   are transposed, if necessary, to give the standard order: longitude, latitude.
      * - Point2Endpoint must be associated with an ast::Frame (not a subclass),
      *   because Frame is the only kind of Frame that is sure to be Cartesian.
      *
      * @param[in] frameSet  ast::FrameSet describing the desired transformation in the usual way:
      *                      from "base" frame to "current" frame
-     * @param[in] simplify  Simplify the frame set? This simplifies each mapping
-     *                      in the frame set by combining component mappings and removing
-     *                      redundant components where possible. However it
-     *                      does not remove any frames.
+     * @param[in] simplify  Simplify the mapping? This combines component mappings
+     *   where it possible to do so without affecting accuracy.
```

Comment:
`where it is possible...`

And should it be indented to match the rest of the docstring block?